runtests.jl

```diff
@@ -28,5 +28,10 @@ for exercise in readdir("exercises")
     @info "Testing $exercise"
     Base.include(m, joinpath(exercise_path, "runtests.jl"))
     
+    # Check that all ipynb's are valid JSON
+    # open("$exercise_path/$exercise.ipynb") do f
+    #     string(JSON3.read(f))
+    # end
+
```

Comment:
What do you think about moving this check to the end of the notebook generator script and enabling it?

Reply:
JSON3 isn't used by any of the other packages and I don't know what packages are available in the images, otherwise, yeah, some kind of validation at time of generation would make sense.

Reply:
You can load all packages that you want in CI. Only need to add a link that runs `Pkg.add` or environment files and `Pkg.instantiate`